src/mixins/highlightConfig.js

```diff
@@ -69,6 +77,11 @@ export default {
         this.shapeView.highlight(this.shapeBody, defaultHighlighter);
       }
     },
+    setBorderOutline(borderOutline) {
+      this.currentBorderOutline ? this.shapeView.unhighlight(this.shapeBody, this.currentBorderOutline) : null;
+      this.currentBorderOutline = borderOutline ? JSON.parse(JSON.stringify(borderOutline)) : null;
+      this.currentBorderOutline ? this.shapeView.highlight(this.shapeBody, this.currentBorderOutline) : null;
+    },
```

Comment:
I find this a little hard to read. Can you simply it? You might be able to take advantage of an early return, and replace two of the ternary operators with actual `if` statements.

Reply:
Improved